.github/pr-size.ps1

```
@@ -20,6 +20,14 @@ try {
         exit 0
     }
 
+    $commitMessages = git log --format=%B "$baseBranch..HEAD" 2>&1
+    if ($LASTEXITCODE -eq 0) {
+        if ($commitMessages -match 'skip-pr-size-check') {
+            Write-Host "Found 'skip-pr-size-check' in commit message, skipping PR size check" -ForegroundColor Yellow
```

PSScriptAnalyzer (1.24.0)

[warning] File 'pr-size.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)